e2e/cypress/integration/system_console/ui_and_api/customization_spec.js

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Customization', () => {
+    before(() => {
+        // # Login as sysadmin and visit customization system console page
+        cy.apiLogin('sysadmin');
+        cy.visit('/admin_console/site_config/customization');
+    });
+
+    it('SC20336 - Can change Custom Brand Image setting', () => {
+        // # Set Enable Custom Branding to true to be able to upload custom brand image
+        cy.get('[id="TeamSettings.EnableCustomBrandtrue"]').check();
+
+        // * Verify that setting is visible and matches text content
+        cy.get('.remove-image').closest('.form-group').find('label').should('be.visible').and('have.text', 'Custom Brand Image:');
+
+        // * Verify that help setting is visible and matches text content
+        const contents = ['Customize your user experience by adding a custom image to your login screen. Recommended maximum image size is less than 2 MB.'];
+        cy.get('.remove-image').closest('.form-group').find('.help-text').should('be.visible').and('have.text', contents[0]);
+
+        // # upload the image
+        cy.fileUpload('input[type="file"]');
+
+        // # save image
+        cy.get('#saveSetting').click();
+
+        // # Verify that after page reload image exist
+        cy.reload();
+        cy.get('img[alt="brand image"]').should('have.attr', 'src').and('include', 'http://localhost:8065/api/v4/brand/image?t=');
```

Comment:
`http://localhost:8065/` will fail on test server other than local, so just test relative path. This can also be wrapped as:
```javascript
cy.findByTestId('CustomBrandImage').should('be.visible').within(() => {
    cy.get('img').should('have.attr', 'src').and('include', '/api/v4/brand/image?t=');
});
```